elixir/advanced-guides/sqlite3.html.md

```diff
@@ -0,0 +1,252 @@
+
+<p class="callout">
+This copies the table structure, table data, indexes, and foreign keys from the MySQL database to the PostgreSQL database.
+<br />
+<br />
+Note that the support for copying is fairly limited. It doesn’t handle database views, functions, triggers, schemas, partial indexes, functional indexes, and many other things. Also, the data type conversion may not be exactly what you want. It is best designed for quick conversions and testing. For serious production use, use the database’s tools to copy databases for the same database type, and for different database types, use the Sequel API.
+</p>
+
+So YMMV on how useful this is for you. If you have a relatively simple database it might work great!
+
+Install ruby if you don't have ruby running on your machine:
+
+```sh
+# Macos
+brew install ruby
+
+# Ubuntu
+apt-get install ruby-full
+```
+
+The ruby database adapter Sequel comes with a really slick command line tool for copying databases, lets install that
+```sh 
+gem install sequel 
+```
+
+Depending on your database you may need to install an extra adapter in this example we will be using Postgres so lets install that:
+```sh
+gem install pg
+```
+
+Next all we need to do is run the Sequel Copy command, where the first database is from and the second database is to
+
+```sh
+sequel -C postgres://localhost/database sqlite3://name.db
+```
+
+And that's it! If you open up your SQLite database using the command line it should have your tables and data all moved over. If you get an error here about connecting you will need to figure out which ruby gem handles your adapter and install it like we did for pg. 
+
+```sh
+sqlite3 name.db
+```
+
+#### Note on types
+
+SQLite doesn't have official support for Postgres Arrays or Hstores, and most special datatypes! These will be copied into strings in the resulting SQLite table, so for Arrays the data will look like `{item1, item2, item3}` if you want to still use this as an array you need to use [string manipulation](https://www.sqlitetutorial.net/sqlite-replace-function/) to convert them to json and then it should work just fine. This is just one example, but the positive is that it's all just strings in SQLite, so if you can make the string look like json you are set! 
+
+### Copying an existing database to a fly volume
+
+If you've exported or copied your database to SQLite you will need to get your database file up to Fly. To do this we will use the [flyctl sftp](/docs/flyctl/sftp/) command. 
```

Comment:
```suggestion
If you've exported or copied your database to SQLite you will need to get your database file up to Fly.io. To do this we will use the [fly sftp](/docs/flyctl/sftp/) command. 
```

We haven't done this retroactively (very much) yet, but the idea is that we'll use `fly` commands in all our new docs.

You can't know this without the style guide, but when we're talking about a feature name, we use "Fly" in front of it (Fly Machines) but otherwise it's "Fly.io".